### PR TITLE
Add file exists functionality to stdlib

### PR DESCRIPTION
**Pull request by snudler6**
Changes: ...

- [ ] Changes fully completed + unit tested + integration tested + documented

### CODE REVIEW

File: stdlibs/os.5.lm [hunk 1]
```diff
@@ -1,5 +1,42 @@
...
+; **************************************
+
+native include "unistd.h"
+
+native code "
+int const R_OK_e = R_OK;
+int const W_OK_e = W_OK;
+int const X_OK_e = X_OK;
+int const F_OK_e = F_OK;
+"
```
Comment (meircif) [inline stdlibs/os.5.lm]: can be removed?

File: stdlibs/os.5.lm [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+    user Array{Char} bla
+    bla := str
+    ptr.set-from-array(user bla)
+
+func exists(user String path)->(var Bool exist)
+    var cdef.Pointer{Char} c-path
+    c-path := string-to-chars(user path)
+    var Int res
+    res := access(copy c-path, copy F-OK)
+    exist := res <> -1
```
Comment (meircif) [inline stdlibs/os.5.lm]: can be only:
`exists := access(copy string-to-chars(user path), copy F-OK) <> -1`

File: stdlibs/tests/unistd.h [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
+#define F_OK 0
+#define R_OK 1
+#define W_OK 2
+#define X_OK 3
+
+int access(char * path, int amode) {
+  return 0;
```
Comment (meircif) [inline stdlibs/tests/unistd.h]: test `path` is `/some/path` and `amode` is `F-OK`

Comment (snudler6) [reply stdlibs/tests/unistd.h]: Validated

File: stdlibs/os.5.lm (hunk 2 above)
Comment (snudler6) [reply stdlibs/os.5.lm]: Changed

File: stdlibs/os.5.lm (hunk 1 above)
Comment (snudler6) [reply stdlibs/os.5.lm]: Removed

**Pull request merged**